src/comp_loinc/groups/groups_builders.py

```diff
@@ -159,6 +160,20 @@ def setup_builder(self, builder):
     builder.cli.command('groups-index-props', help='Indexes the use of LOINC properties by property and part.')(
         self.__main)
 
+  def parts_to_tsv(self, parts: t.List[Part]):
```

Comment:
This is general; maybe you already had a general func for this?

<details><summary>Details</summary>
<p>

Or  perhaps this would be better as a separate func.

I was going to make a `dangling_to_tsv()` but I did this instead:
```py
    dangling: t.Dict[str, Part] = self.index.parts_roots_no_children
    self.parts_to_tsv(list(dangling.values()))
```

Probably out of a love of pure(er) functions. Doing it the OO way involved 6 lines of checking for potentially non-existent properties.

</p>
</details>